roles/matrix-maubot/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+matrix_maubot_enabled: true
```

Comment:
This is a good default for the role itself (for people who would include it in their own other playbook, especially), but it'd be good if we also adjust `group_vars/matrix_servers` to:

- disable this by default (like we do for other bots and bridges): `matrix_maubot_enabled: false`
- override some other configuration defaults that makes sense in the context of the playbook (probably unnecessary as of right now)